app/src/main/java/com/example/trainingintive/presentation/SplashViewModel.kt

```diff
@@ -41,4 +42,14 @@ class SplashViewModel @Inject constructor(
     fun onFailedLogin() {
         navigator.sendEvent(SplashScreenEvent.Error)
     }
+
+    private fun checkIfUserIsLoggedAndSendEventToNavigator() {
+        if (isUserLogged()) {
+            navigator.sendEvent(SplashScreenEvent.DisplayUserScreen)
+        } else {
+            navigator.sendEvent(SplashScreenEvent.DisplayLogInForm)
+        }
+    }
+
+    private fun isUserLogged() = FirebaseAuth.getInstance().currentUser != null
```

Comment:
It should not be in this layer. It will be best to return this information to viewmodel via usecase and repository. You can leave it for now, but add TODO for this.